src/IdeaStatiCa.IntermediateModel/IRModel/SList.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace IdeaStatiCa.IntermediateModel.IRModel
+{
+	// List intermediate object
+	public class SList : ISIntermediate
+	{
+		public ICollection<ISIntermediate> Items { get; set; }
+		public int Count { get { return Items.Count; } }
+
+		public SList()
+		{
+			Items = new List<ISIntermediate>();
+		}
+
+		public SList(ISIntermediate item)
+		{
+			if (item == null)
+			{
+				Items = new List<ISIntermediate>();
+			}
+			else
+			{
+				Items = new List<ISIntermediate>() { item };
+			}
+		}
+
+		public SList(params ISIntermediate[] items)
+		{
```

Comment:
Having method with params makes the method above redundant. 
=> That would be same as this with one parameter

Reply:
remove for one param